### ISSUE TITLE
Configuration GUI does not show everything

### ISSUE THREAD

**Issue by aybe** [joncampbell123/dosbox-x#738]
I've been trying to look at fixing mainly the overflowing labels, using bigger windows and use a bunch of macros on top of the file so all the sizes can be changed at once and one doesn't have to go through the code.

So far that was promising until I found out that some of the section are not showing everything, 720*400 is simply not enough to draw all the options in the 'log'. While with this new approach I could fit 'log' a real fix would be a scroll viewer control of some sort but honestly, this is out of reach to me.

So is it deprecated in favor of the new SDL2 menu or will it stay ? i.e. is it worth trying what I had in mind ?

old version:
![2018-05-25 19_19_36-window](https://user-images.githubusercontent.com/1537591/40557874-de954d48-6051-11e8-852c-313993af6cc0.png)

preliminary new version:
![2018-05-25 19_28_17-window](https://user-images.githubusercontent.com/1537591/40557888-ee466aba-6051-11e8-91f5-7001bb8c8e9d.png)

see how many are still hidden:
![2018-05-25 19_28_25-window](https://user-images.githubusercontent.com/1537591/40557890-f0e6dea8-6051-11e8-981e-9fb57e0d7f47.png)

**Comment by joncampbell123** [comment]
I think the solution is to modify the GUI_TK library to add scrollbars and scrolling.

Then it can either show all the options or put it into a subregion and scroll the options.

The second screenshot however shows that the layout code needs to make better use of the vertical space in the dialog box, which would help all the options fit on the screen.

**Comment by aybe** [comment]
EDIT:

Even the help windows are lacking space, we could edit their text but it's a poor approach IMO.

![2018-05-25 19_39_37-window](https://user-images.githubusercontent.com/1537591/40558371-69d72b50-6053-11e8-9f9f-27bab6c78473.png)

**Comment by aybe** [comment]
Yes, the thing is it looks daunting to me :) I'll check out, maybe there has been an update to the library which could save from doing this.

**Comment by aybe** [comment]
Thought it was an external library but doesn't seem to be the case ...

**Comment by aybe** [comment]
Well ... again, I tried to take a look but it's absolutely insane how the code is intertwined, clearly I don't have the skills :( I'll see if I can do something else I can reach !
